config/default.yaml

```
@@ -234,3 +234,7 @@ rabbitmq:
     prefetchCount: "20"
     # Message reprocess limit - max retry for a message
     messageReprocessLimit: "3"
+httpd:
+    port: 80
+    host: 0.0.0.0
+    uri: http://IP_ADDRESS:PORT
```

Same as above (e.g. [API](https://github.com/screwdriver-cd/screwdriver/blob/cda501eeaab14d8833b983febe8051624c33652a/config/default.yaml#L49-L55))
```suggestion
    # Port to listen on
    port: 80
    # Host to listen on (set to localhost to only accept connections from this machine)
    host: 0.0.0.0
    # Externally routable URI (usually your load balancer or CNAME)
    uri: http://IP_ADDRESS:PORT
```